x/launch/keeper/msg_server_create_chain_test.go

```diff
@@ -79,6 +90,12 @@ func TestMsgCreateChain(t *testing.T) {
 
 			// Chain created from MsgCreateChain is never a mainnet
 			require.False(t, chain.IsMainnet)
+
+			if tc.msg.HasCampaign {
```

Comment:
We can add the check `require.False(t, chain.HasCampaign)` and `require.Equals(t, tc.msg.CampaignID, chain.CampaignID)`